### PR TITLE
Fix to work split mode properly with xliff 2.0 format 

### PR DESCRIPTION
**Pull request by gouniLee**
* Fix to work `split`  mode properly with xliff 2.0 format and separate to function for testing
* Add unit test to verify split mode
* regarding xliff 2.0 case, the project split type is only valid.
usage:
`node loctool.js split project test/testfiles/xliff20/merge-en-US.xliff -2 --targetDir output`

result would be like:
```
 output/app1/en-US.xliff
 output/app2/en-US.xliff
```

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -251,41 +251,45 @@ limitations under the License.
         <run script="${build.test}/testXliffMerge.js" executable="${nodeunit}/bin/nodeunit" dir="${build.test}" args="" />
     </target>
 
+    <target name="test.xliffsplit" description="Run only the xliff merge tests">
```
Comment (ehoogerbeets) [inline build.xml:254]: Maybe next time, please do the whitespace fixing in a separate PR, as it hides this change.

**Comment by ehoogerbeets** [comment]
Since this didn't make the cut for the last loctool release, please update the version number to 2.10.1 and update the release notes as well. Thanks!

File: build.xml (hunk 1 above)
Comment (gouniLee) [reply build.xml:254]: Yep, I'll seperate it next time

**Comment by gouniLee** [comment]
> Since this didn't make the cut for the last loctool release, please update the version number to 2.10.1 and update the release notes as well. Thanks!

Yeah.. It's late to include the version 2.10.0. I fixed it.

**Review by ehoogerbeets — APPROVED** [review]
